e2e/config.js

```diff
@@ -40,6 +41,10 @@ module.exports = {
         expectedPrompt: /What is your SLAS Client ID?/i,
         response: "987fc116-d30c-4537-93cb-c2bd433c3b5a\n",
       },
+      {
+        expectedPrompt: /Is your SLAS client private?/i,
+        response: "2\n",
```

Comment:
I assume 2 in this case means no and that it corresponds to a public client?

Reply:
Correct!